front/app/modules/commercial/insights/api/categories/keys.ts

```diff
@@ -0,0 +1,17 @@
+const categoriesKeys = {
+  all: () => [{ type: 'category' }] as const,
+  lists: () => [{ ...categoriesKeys.all()[0], entity: 'list' }] as const,
+  list: (viewId: string) =>
+    [{ ...categoriesKeys.all()[0], entity: 'list', viewId }] as const,
+  details: () => [{ ...categoriesKeys.all()[0], entity: 'detail' }] as const,
+  detail: (viewId: string, id: string) =>
+    [
+      {
+        ...categoriesKeys.details()[0],
+        viewId,
+        id,
+      },
+    ] as const,
+};
```

Comment:
Yes, good point. I see how the naming might be confusing.. I would go with `operation` because we have `action` in the fetcher and it refers to the request type - `get`, `post`, etc. so it might be confusing.